### DISCUSSION TITLE
Unable to remove non-admin users from Teams dashboard

### DISCUSSION THREAD

**Discussion by rfreebern** [codemagic-ci-cd/codemagic-docs#2795] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

1. Open "Team settings" in the Codemagic web app.
2. Expand the "Users" list.
3. Click the three dots next to a non-admin user.
4. Click the "Remove user" link in the dropdown that appears.

### Expected results

I expect a confirmation dialog to appear which will then allow me to remove the user.

### Actual results

Clicking the "Remove user" link has no effect. No errors are logged in the browser console.

**Comment by dam-ease** [comment]
Hello @rfreebern,
To confirm I got the issue you are currently experiencing, do you mean that you don't see the dialog below after clicking the remove user button in the dropdown?

<img width="608" alt="Screenshot 2024-10-23 at 09 36 06" src="https://github.com/user-attachments/assets/631cfefb-48b2-45c2-aa7e-e88112f28ff0">

Also, does the user get removed after clicking the remove user button in the dropdown?
Please can you share your Team ID here with me, you can get it in your team settings (see screenshot)
<img width="1202" alt="Screenshot 2024-10-23 at 09 38 28" src="https://github.com/user-attachments/assets/e7977722-16cf-4d1e-954e-8660526e7b9d">

**Reply by dam-ease** [reply, marked as answer]
Hello @rfreebern,
We discovered that this issue was happening on some browsers and it has been fixed now, please can you try this again to confirm it works fine on your end now?

**Reply by rfreebern** [reply]
Thank you! I can confirm that it's working properly now.
